boards/common/arduino-zero/include/arduino_iomap.h

```diff
@@ -64,11 +64,26 @@ extern "C" {
  * @name    Mapping of MCU pins to Arduino pins
  * @{
  */
+
+/* Arduino Zero clones have D2 and D4 swapped compared to official boards, so
+ * we have handle them differently here. */
+#ifdef BOARD_WEMOS_ZERO
```

Comment:
Is Wemos Zero the only Arduino Zero clone that we support?

Reply:
Then maybe change the comment from "Arduino Zero clones have" to "Wemos Zero has"?

Reply:
or "Wemos Zero boards have"

Reply:
I rephrased the comment, fixed some sloppy copy-paste error, and squashed right away.